solution.txt

```diff
@@ -192,7 +195,14 @@ Exercise 14:
 
 // Complete Exercise 14 below...
 
-const num =numArrays[2][1];
+var x=[]
+for (let i=0; i<numArrays.length; i++)
+{x=numArrays[i]
+for (let i=0; i<x.length; i++)
+  {if (x[i] === 66) 
+  {var num=x[i]}
+  }
+}
```

Comment:
you're doing something similar as in line 181ish here. the opening curly bracket of a loop or conditional should always be in the same line as the loop or conditional. see lines 199/200, 201/202, 202/203 for the places where you put the opening curly bracket in the wrong spot